Croatian Army Woodland loadout

TheLustyDragon opened the pull request:
Added loadout.
Doesn't contain UAV operator.

beaar commented:
Every role should be covered by the loadout. You need to include the UAV operator (and if you removed any other roles, them too) loadout even if you can't find appropriate equipment for it.

You also need an entry in the CfgLoadouts.hpp file that links the loadout. Look at how the other loadouts are linked and do the same for yours.
https://github.com/Sniperhid/1tac_misc/blob/master/addons/tm_tmf_loadouts/CfgLoadouts.hpp

TheLustyDragon commented:
Properly uploaded loadout to loadouts folder.
Updated CfgLoadouts.hpp to include the new loadout.
Updated loadout to mention me as author.
Loadout now also contains the UAV operator, no other classes were removed.

I think I did it right this time.

beaar commented:
Waiting for the second loadout

TheLustyDragon commented:
Added the 1992 loadout entry to CfgLoadouts.hpp and added tooltips to both loadouts.

beaar commented:
There's a lot of issues with the 1992 loadout. Vest and uniforms that can't fit magazines etc. You should test the loadouts more thoroughly. Run TMF Autotest in the editor.

beaar commented on addons/tm_tmf_loadouts/loadouts/cro_army_1992.hpp:
```diff
@@ -58,13 +58,14 @@ class r : baseMan
     };
 };
-class car : r
+class car
```
Why did you remove the parent class?

beaar commented:
The "armed suits" loadouts are both missing a CfgLoadouts.hpp entry. They probably belong in the 'Other' category.

beaar commented:
I want you to put the suits loadouts in the 'Other' category since they don't really have any national identity.

beaar commented on addons/tm_tmf_loadouts/loadouts/cro_army_1992.hpp:
```diff
@@ -0,0 +1,441 @@
+	backpack[] = {"rhs_rpg_empty"};
+    backpackItems[] =
+    {
+        LIST_2("rhs_mag_9k38_rocket ")
```
trailing space

beaar commented:
You need to test the loadouts more thoroughly yourself, so we don't have to go back and forth like this, wasting both my and your time. There are a lot of autotest errors and at least one popup error because of the trailing space I highlighted in the earlier comment.
